Web/starter_project_vanguard/components/layout/footer.tsx

```diff
@@ -0,0 +1,112 @@
+import Link from 'next/link'
+import { FaFacebook, FaInstagram, FaLinkedin, FaTwitter, FaYoutube} from 'react-icons/fa'
+import inpowered from '../../public/images/inpowered.png'
+import Image from 'next/image'
+import { NavItem } from '@/types/nav'
+
+const navLinks: NavItem[] = [
+  { name: 'Home', href: '/' },
+  { name: 'Success Stories', href: '/story' },
+  { name: 'About Us', href: '/about' },
+  { name: 'Get Involved', href: '/get-involved' },
+]
+
+const navTeams: NavItem[] = [
+  { name: 'Board Members', href: '/board' },
+  { name: 'Advisors/Mentors', href: '/advisors' },
+  { name: 'Executives', href: '/executives' },
+  { name: 'Staffs', href: '/staffs' },
+]
+
+const navBlogs: NavItem[] = [
+  { name: 'Recent Blogs', href: '/recent-blogs' },
+  { name: 'New Blog', href: '/new-blog' },
+]
+
+const Footer: React.FC = () => {
+  return (
+    <footer className="mt-auto border-y border-text-secondary px-12">
+      <div className="flex justify-between py-12 items-center space-x-12">
+        <div className="basis-1/5">
+          <Link href="/">
+            <Image src={inpowered} alt="inpowered logo" />
+          </Link>
+        </div>
+
+        <div className="basis-1/3">
+          <h3 className="text-lg font-semibold my-6">
+            Get involved in improving tech education in Africa!
+          </h3>
+          <button className="bg-primary text-white rounded-md w-[190px] h-[45px]">
```

Comment:
is there maybe a class name to define botton large or button small or sth? I would prefer to have that instead of stating the specific pixels (it would help to stay consistent with others)

it might be worth checking

Reply:
[Previous comment](https://github.com/RealEskalate/Internship-2023/pull/39#discussion_r1174620965)